src/components/Rewards/AmountHeader.vue

```diff
@@ -66,15 +67,18 @@ $divider-height: 40px;
     }
 
     &__amount {
+      color: white;
       font-size: $amount-font-size;
       font-weight: $s-font-weight-big;
-      line-height: $s-line-height-medium;
+      line-height: $amount-line-height;
       letter-spacing: $s-letter-spacing-big;
     }
 
     &__symbol {
+      color: white;
```

Comment:
maybe we can use `white` from js-ui to easily replace it when dark theme will be ready?